src/storage-preview/azext_storage_preview/_params.py

```diff
@@ -127,8 +127,22 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help=' Indicate the number of days that the deleted item should be retained. The minimum specified '
                         'value can be 1 and the maximum value can be 365.')
         c.argument('enable_smb_multichannel', options_list=['--enable-smb-multichannel', '--mc'],
-                   arg_type=get_three_state_flag(), min_api='2020-08-01-preview',
+                   arg_type=get_three_state_flag(), min_api='2020-08-01-preview', arg_group='SMB Setting',
                    help='Set SMB Multichannel setting for file service. Applies to Premium FileStorage only.')
+        c.argument('versions', arg_group='SMB Setting', min_api='2020-08-01-preview',
+                   help="SMB protocol versions supported by server. Valid values are SMB2.1, SMB3.0, "
+                        "SMB3.1.1. Should be passed as a string with delimiter ';'.")
+        c.argument('authentication_methods', options_list='--auth-methods', arg_group='SMB Setting',
+                   min_api='2020-08-01-preview',
+                   help="SMB authentication methods supported by server. Valid values are NTLMv2, Kerberos. "
+                        "Should be passed as a string with delimiter ';'.")
+        c.argument('kerberos_ticket_encryption', options_list=['--kerb-ticket-encryption', '-k'],
+                   arg_group='SMB Setting', min_api='2020-08-01-preview',
+                   help="Kerberos ticket encryption supported by server. Valid values are RC4-HMAC, AES-256. "
+                        "Should be passed as a string with delimiter ';'.")
+        c.argument('channel_encryption', arg_group='SMB Setting', min_api='2020-08-01-preview',
+                   help="SMB channel encryption supported by server. Valid values are AES-CCM-128, AES-GCM-128, "
+                        "AES-GCM-256. Should be passed as a string with delimiter ';'.")
```

Comment:
Do these valid values have enum definition in SDK? Can we add format validator for these `SMB Setting`?

Reply:
No. They are not one specific value. The accepted value is a combination of several valid values with `,`.

Reply:
I see. By format validator, I mean splitting user input and check whether each value is valid(within enum options). But if there's no enum in SDK, then fine with me.